sql/updates/world/3.3.5/2016_05_24_00_world_335.sql

```diff
@@ -0,0 +1,682 @@
+RENAME TABLE `game_graveyard_zone` TO `graveyard_zone`;
+ALTER TABLE `graveyard_zone` CHANGE `id` `ID` MEDIUMINT(8) UNSIGNED NOT NULL DEFAULT '0';
+ALTER TABLE `graveyard_zone` CHANGE `ghost_zone` `GhostZone` MEDIUMINT(8) UNSIGNED NOT NULL DEFAULT '0';
+ALTER TABLE `graveyard_zone` CHANGE `faction` `Faction` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0';
+ALTER TABLE `graveyard_zone` ADD COLUMN `Comment` TEXT AFTER `Faction`;
+
+UPDATE `command` SET `help`='Syntax: .reload all area\n\nReload areatrigger_teleport, areatrigger_tavern, graveyard_zone tables.' WHERE `permission`=613;
+UPDATE `command` SET `name`='reload graveyard_zone', `help`='Syntax: .reload graveyard_zone\nReload graveyard_zone table.' WHERE `permission`=645;
+
+UPDATE `graveyard_zone` SET `Comment`='Redridge Mountains - Redridge Mountains' WHERE `ID`=2;
+UPDATE `graveyard_zone` SET `Comment`='Duskwood, Darkshire - Duskwood' WHERE `ID`=3;
+UPDATE `graveyard_zone` SET `Comment`='Westfall, Sentinel Hill GY - Westfall' WHERE `ID`=4;
+UPDATE `graveyard_zone` SET `Comment`='Loch Modan, Thelsamar - Loch Modan' WHERE `ID`=6;
+UPDATE `graveyard_zone` SET `Comment`='Wetlands, Crossroads GY - Dun Morogh' WHERE `ID`=7;
+UPDATE `graveyard_zone` SET `Comment`='Wetlands, Crossroads GY - Wetlands' WHERE `ID`=7;
+UPDATE `graveyard_zone` SET `Comment`='Wetlands, Crossroads GY - Loch Modan' WHERE `ID`=7;
+UPDATE `graveyard_zone` SET `Comment`='Badlands, Graveyard NE - Badlands' WHERE `ID`=8;
+UPDATE `graveyard_zone` SET `Comment`='Badlands, Graveyard NE - Loch Modan' WHERE `ID`=8;
+UPDATE `graveyard_zone` SET `Comment`='The Barrens, The Crossroads - Northern Barrens' WHERE `ID`=10;
+UPDATE `graveyard_zone` SET `Comment`='Desolace, Ethel Rethor GY - Desolace' WHERE `ID`=31;
+UPDATE `graveyard_zone` SET `Comment`='Desolace, Ethel Rethor GY - Maraudon' WHERE `ID`=31;
```

Comment:
You replace comment above. Should use in query GhostZone:

UPDATE `graveyard_zone` SET `Comment`='Desolace, Ethel Rethor GY - Desolace' WHERE `ID`=31 AND `GhostZone` = 405;
UPDATE `graveyard_zone` SET `Comment`='Desolace, Ethel Rethor GY - Maraudon' WHERE `ID`=31 AND `GhostZone` = 2100;